test/unit/lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -395,6 +395,13 @@ describe('AwsInvokeLocal', () => {
       await awsInvokeLocal.loadEnvVars();
       expect(process.env.providerVar).to.be.equal('providerValueOverwritten');
     });
+
+    it('it should not pass null env vars to handler', async () => {
```

Comment:
All new tests should we written with `runServerless` utility - more details available here: https://github.com/serverless/serverless/blob/master/test/README.md 

You can take a look at this one to serve as an example on how to write a similar test for introduced fix: https://github.com/serverless/serverless/blob/7072795a50ce87879089d41335641ac37d4abd09/test/unit/lib/plugins/aws/invokeLocal/index.test.js#L1542

Reply:
I tried to rewrite the test with runServerless.